tasks/promtail.yml

```diff
@@ -25,17 +25,9 @@
     mode: 0644
     force: yes
 
-- name: Create Systemd service
-  block:
-    - name: Template the new service file for Promtail agent
-      template:
-        src: promtail.service.j2
-        dest: /etc/systemd/system/promtail.service
-        mode: '0644'
-
-    - name: Reload and start service Promtail agent
-      service:
-        name: promtail
-        state: restarted
-        daemon_reload: yes
-        enabled: yes
\ No newline at end of file
+- name: Create Promtail's Supervisor config file
```

Comment:
If useing Supervisor, I guess we need a handler to re(start) the service under supervisor? See https://github.com/GameAnalytics/ansible-collect/blob/master/handlers/main.yml for example

Reply:
Not in our case, as we only provision a container which is being ran by fargate. 

I defined our entrypoint to start supervisord, but alternatively one could define a post-task section in their ansible play which runs after the role.

Reply:
Not in our case, as we only provision a container which is being ran by fargate. 

I defined our entrypoint to start supervisord, but alternatively one could define a post-task section in their ansible play which runs after the role.

Reply:
Yes, makes sense... the role is only used for provisioning in our case (runs only once) so indeed it does not matter... but being a public role and so on could be the case someone uses it in a different way...

All our roles are used only for provisioning so executed only once...

Reply:
Okay since that is exactly what we need, i guess it doesnt hurt our infrastructure.

> When state = present, the module will call supervisorctl reread then supervisorctl add if the program/group does not exist.
> When state = restarted, the module will call supervisorctl update then call supervisorctl restart.

Thanks :)